langlib/langlib-test/src/test/resources/test-src/valuelib_test.bal

```diff
@@ -50,6 +50,33 @@ function testToJsonString() returns map<string> {
     return result;
 }
 
+function testToJsonStringForNonJsonTypes() {
```

Comment:
Can you also include a couple of tests where,
- the variable type is `anydata`
- the variable type is a record with an `xml` field

Reply:
Sure.